purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -913,6 +918,63 @@ class PurchasesTest {
         assertThat(receivedOfferings!![stubOfferingIdentifier]!!.monthly!!.product).isNotNull
     }
 
+    @Test
+    fun `configuration error if no products are found`() {
```

Comment:
the current errors are logged with `LogIntent.GOOGLE_ERROR`, i wonder if we should be using something else...

Reply:
Oooh good catch. I added a proposed solution but let me know if we should do a less localized fix, maybe putting the logic in `PurchasesErrorCode`.

Reply:
nice 👍  i think what you added is good for now... i think we have some error cleanup to do on both natives that probably needs some discussion anyway.